conceptnet5/uri.py

```diff
@@ -181,7 +100,7 @@ def concept_uri(lang, text, pos=None, disambiguation=None):
         ...
     AssertionError: 'this is wrong' is not in normalized form
     """
-    assert text == standardize_text(text), "%r is not in normalized form" % text
+    assert ' ' not in text, "%r is not in normalized form" % text
```

Comment:
It's just a sanity check. I'm saving time by not re-running the actual standardization, just assuming that if you're not standardizing the text, at some point you'll pass in something with a space in it.